ISSUE_TEMPLATE.md

```diff
@@ -33,3 +33,7 @@ You can find this information by going to the navigation drawer in the app and t
 **Screen-shots:** 
 
 Can be created by pressing the Volume Down and Power Button at the same time on Android 4.0 and higher.
+
+**Would you like to work on the issue?**
+
+Please let us know if you can work on it or the issue should be assigned to someone else.
```

Comment:
These days many issues are submitted by developers, but most of the time issues are submitted by users who don't know anything about programming and might misunderstand this as "I can help by providing feedback". So I suggest changing to the following and then merging:

Please let us know whether you want to fix the issue by yourself. If not, anyone can get the issue assigned to them.

Reply:
Yeah, I agree with this. Happy to merge after this change is made. :)

Reply:
I had updated it.